cluster/manifests/kube-static-egress-controller/deployment.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: kube-static-egress-controller
+  namespace: kube-system
+  labels:
+    application: kube-static-egress-controller
+    version: v0.0.3
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      application: kube-static-egress-controller
+  template:
+    metadata:
+      labels:
+        application: kube-static-egress-controller
+        version: v0.0.3
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+        iam.amazonaws.com/role: "{{ .LocalID }}-static-egress-controller"
+    spec:
+      serviceAccountName: system
```

Comment:
The controller only needs to read from the API right? In that case I would drop this and just use the default service account which has readOnly access.